kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,80 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+def get_pod_ready_status():
+    pods_status = runcommand.invoke("kubectl get pods -n litmus -o yaml")
+    yaml_pod = yaml.full_load(pods_status)
+    logging.info('len of pod items' + str(len(yaml_pod['items'])))
+    for pod in yaml_pod['items']:
+        return pod['status']['phase']
+
+
+# Install litmus and wait until pod is running
+def install_litmus():
+    runcommand.invoke("kubectl apply -f "
+                      "https://litmuschaos.github.io/litmus/litmus-operator-v1.8.2.yaml")
+
+    # wait until pods are running
+
+    ready_status = get_pod_ready_status()
+    logging.info("first ready status for pod" + str(ready_status))
+    while ready_status != "Running":
+        time.sleep(10)
+        logging.info("ready status for pod" + str(ready_status))
+        ready_status = get_pod_ready_status()
+        if "Error" in ready_status:
+            logging.error("Litmus pod had error creating")
+            sys.exit(1)
+
+
+# Check status of experiment
+def check_experiment(engine_name, experiment_name, namespace):
+    chaos_engine = runcommand.invoke("kubectl describe chaosengines/%s -n %s | grep "
+                                     "Status" % (engine_name, namespace))
+    engine_status = chaos_engine.split(": ")[-1].strip()
+    while engine_status != "Running" and engine_status != "Completed":
```

Comment:
We should have a max_retry count here.  I've seen cases where the experiment or engine are mis-configured and we just wait for ever.